tla/consensus/Network.tla

```diff
@@ -78,28 +78,28 @@ LOCAL ReorderNoDupDropMessages ==
 \* Point-to-Point Ordering and duplication of messages:
 
 LOCAL OrderInitMessageVar ==
-    messages = [ s \in Servers |-> <<>>]
+    messages = [ src \in Servers |-> [ dst \in Servers |-> <<>> ] ]
 
 LOCAL OrderWithMessage(m, msgs) ==
-    [ msgs EXCEPT ![m.dest] = Append(@, m) ]
+    [ msgs EXCEPT ![m.source][m.dest] = Append(@, m) ]
 
 LOCAL OrderWithoutMessage(m, msgs) ==
-    [ msgs EXCEPT ![m.dest] = RemoveFirst(@, m) ]
+    [ msgs EXCEPT ![m.source][m.dest] = RemoveFirst(@, m) ]
 
 LOCAL OrderMessages ==
-    UNION { Range(messages[s]) : s \in Servers }
+    UNION { UNION { Range(messages[src][dst]) : dst \in Servers } : src \in Servers }
```

Comment:
@lemmy The rest of these changes are reasonably mechanical, so I assume if one is right then they're all right. This change, and `OrderedMessagesTo`, are more involved - can you check that they're doing what was originally intended?